---
title: "'Příkaz catch' bloku nedospělo, protože '&lt;name1&gt;\"dědí z\"&lt;name2&gt;\""
ms.date: 07/20/2015
f1_keywords:
- BC42029
- vbc42029
helpviewer_keywords:
- BC42029
ms.assetid: 4bc84a84-ec7e-49db-8794-4fb71d36d5c0
ms.openlocfilehash: 7a51857bf2f181224c64a4295001e5ea24474cdb
ms.sourcegitcommit: 0888d7b24f475c346a3f444de8d83ec1ca7cd234
ms.translationtype: MT
ms.contentlocale: cs-CZ
ms.lasthandoff: 12/22/2018
ms.locfileid: "53767475"
---
# <a name="catch-block-never-reached-because-ltname1gt-inherits-from-ltname2gt"></a>'Příkaz catch' bloku nedospělo, protože '&lt;name1&gt;"dědí z"&lt;name2&gt;"
A `Catch` nedospělo bloku kódu.  
  
 Ve výchozím nastavení tato zpráva je upozornění. Další informace o zobrazení nebo skrytí upozornění zpracování upozornění jako chyby, najdete v části [Konfigurace upozornění v jazyce Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).  
  
 **ID chyby:** BC42029  
  
## <a name="to-correct-this-error"></a>Oprava této chyby  
  
-   Odebrat cizí `Catch` bloku.  
  
## <a name="see-also"></a>Viz také  
 [Postupy: Testování kódu pomocí bloku Try... Blok catch v jazyce Visual Basic](https://msdn.microsoft.com/library/8368e205-ed73-4185-a247-af84fb4fafa9)  
 [Příkaz Try...Catch...Finally](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
